### PR TITLE
Add VibeVoice Realtime TTS export

### PR DESCRIPTION
**Pull request by justinchuby**
## Summary

- Adds a source-faithful six-stage Mobius export for `microsoft/VibeVoice-Realtime-0.5B` at `6bce5f06044837fe6d2c5d7a71a84f0416bd57e4`: embedding, the 4-layer lower LM backbone, 20-layer TTS backbone, speech connector, diffusion velocity head, and causal audio decoder.
- Exposes all KV and 34 causal-convolution states; host code owns 5-token streaming windows, cached voice presets, classifier-free guidance, DPM-Solver, and stopping.
- Pins the Microsoft reference package source to `79e516a3e20b599f137c9da03410a2a0b473b63b`, adds source parity/continuation coverage, full-config validation, processor-contract coverage, exhaustive weight routing, revision-pin protection, and honest runtime metadata.
- Uses a causal prefix-valid BOOL attention mask, enabling normal CUDA FP16 GQA lowering without arbitrary-mask fusion.

## Validation

- Affected regression suite: 1,117 passed, 278 skipped, 1 xfailed.
- Focused Realtime config/graph/source-parity/continuation/L2 validation: 14 passed.
- Full float16 and float32 real-weight exports completed. A Carter cached-prompt, one five-token window with two source DPM-Solver iterations and one causal decode produced 3,200 finite samples (0.133333 s): FP16 CPU 1.915602 s / RTF 14.3670; MLX 0.733651 s / RTF 5.5024, with max CPU/MLX waveform delta `9.536743e-7`; FP32 CPU 5.248908 s / RTF 39.3668.

## Runtime boundaries

- This is deliberately host-orchestrated; ORT GenAI configuration fails closed and ONNX GenAI metadata is advisory-only.
- Native BF16 graph serialization succeeds, but stock ORT 1.29 CPU lacks required BF16 kernels. No CUDA provider was available for runtime performance validation. CoreML loaded the FP16 decoder but profiled no CoreML nodes.
- The real-weight probe is one latent, not a full six-latent streaming window or L5 golden-generation assertion.

### CODE REVIEW

File: src/mobius/models/vibevoice_streaming_test.py [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+    import transformers
+    from transformers import AutoModel
+
+    # The source's processor imports the pre-5.16 private Qwen2 tokenizer path,
+    # while its model registration predates an existing native registration.
+    # Neither compatibility shim participates in neural execution.
+    original_register = AutoModel.register
+
+    def register_with_existing_mapping(cls, config_class, model_class, exist_ok=False):
+        return original_register(config_class, model_class, exist_ok=True)
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/models/vibevoice_streaming_test.py]: Fixed in `3cfe0e93` by renaming the intentionally ignored `AutoModel.register` callback parameters to `_cls` and `_exist_ok`. The shim still always delegates with `exist_ok=True`, which is required only to import the pinned historical source against current Transformers. CI-equivalent lint passes.

File: src/mobius/models/vibevoice_streaming.py [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+    def forward(
+        self,
+        op: OpBuilder,
+        inputs_embeds: ir.Value,
+        attention_mask: ir.Value,
+        position_ids: ir.Value,
+        lm_last_hidden_state: ir.Value,
+        tts_text_masks: ir.Value,
+        past_key_values: Sequence[tuple[ir.Value, ir.Value]],
+    ):
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/models/vibevoice_streaming.py:127]: Fixed in `3cfe0e93`. The TTS override now preserves the base positional contract and exposes its source-specific tensors as optional trailing arguments. The task supplies both by keyword through `nn.Module.__call__`, which preserves ONNX parameter naming; a paired-input guard prevents partial use. Focused Realtime tests pass, and the full L1 smoke suite passes (1,601 passed, 52 skipped).
